src/plugins/discover/public/application/doc_views/doc_views_registry.ts

```diff
@@ -6,33 +6,16 @@
  * Side Public License, v 1.
  */
 
-import { auto } from 'angular';
-import { convertDirectiveToRenderFn } from './doc_views_helpers';
 import { DocView, DocViewInput, ElasticSearchHit, DocViewInputFn } from './doc_views_types';
 
 export class DocViewsRegistry {
   private docViews: DocView[] = [];
-  private angularInjectorGetter: (() => Promise<auto.IInjectorService>) | null = null;
-
-  setAngularInjectorGetter = (injectorGetter: () => Promise<auto.IInjectorService>) => {
-    this.angularInjectorGetter = injectorGetter;
-  };
 
   /**
    * Extends and adds the given doc view to the registry array
    */
   addDocView(docViewRaw: DocViewInput | DocViewInputFn) {
     const docView = typeof docViewRaw === 'function' ? docViewRaw() : docViewRaw;
-    if (docView.directive) {
```

Comment:
I think we still need to remove the `DirectiveDocViewInput` from `doc_views_types`.

Reply:
thx, yes! I've changed that, also thx for adding to the PR's description!